source/stylesheets/all.css

```diff
@@ -496,8 +513,18 @@ body.index > .container p:nth-of-type(2) a:hover{
 /* Responsive */
 
 @media (min-width: 1200px) {
+    .navbar-nav a {
+        opacity: .5;
+    }
+    .text-md-right{
+        text-align: right;
+    }
+    footer .nav-pills>li {
+        display: inline-block;
+    }
     header {
         margin: 95px 0 95px;
+        z-index: 1;
```

Comment:
This has made the search box on the tooling section un-usable, I can't get access to input bar since it's blocked by the header.

@tjerabek would you be able to take a look, this is especially noticeable at around 1100px wide (my usually browser width)?